src/framework/auth.rs

```diff
@@ -2,8 +2,10 @@ use reqwest::RequestBuilder;
 
 #[derive(Debug)]
 pub enum Credentials {
-    User { key: String, email: String },
+    UserAuthKey { email: String, key: String },
+    UserAuthToken { token: String },
     Service { key: String },
+    Default,
```

Comment:
See my comment on `example/main.rs` - I don't think we should add a `Default` variant. We don't ever want a user to actually use it, so let's just never implement it in the first place :)

Reply:
Agreed, it was more of a perceived necessary evil at the time :)